packages/backend-test-utils/api-report.md

```diff
@@ -23,13 +24,15 @@ export function setupRequestMockHandlers(worker: {
 export function startTestBackend<
   TServices extends any[],
   TExtensionPoints extends any[],
->(options: TestBackendOptions<TServices, TExtensionPoints>): Promise<void>;
+>(options: TestBackendOptions<TServices, TExtensionPoints>): Promise<Backend>;
 
 // @alpha (undocumented)
 export interface TestBackendOptions<
   TServices extends any[],
   TExtensionPoints extends any[],
 > {
+  // (undocumented)
+  autoStop?: boolean;
```

Comment:
Yep, could convert this to `'afterEach' | 'never'` for now

Reply:
Alright. I just felt it's pretty common to fire up one instance of a thing and doing several tests against it. Whether it's a good idea or not :)

Reply:
Ah, I don't think we can do `'afterAll'` tbh, since it will always be scoped to the entire test file. I'm pretty sure that would lead to confusion if you're using it within a specific `describe` block that you expect it to be scoped to.

Reply:
Will it? You can trigger the afterAll within a describe and it'll apply only within that describe block eh? (Ie after all THOSE tests) Maybe I'm misinformed.

Reply:
Yep confirmed, that's what happens; `before*` and `after*` are "scoped".